indico/modules/events/registration/client/js/form/fields/SingleChoiceInput.jsx

```diff
@@ -355,12 +356,13 @@ export default function SingleChoiceInput({
       required={isRequired}
       isRequired={isRequired}
       disabled={disabled}
-      defaultValue={defaultItem ? {[defaultItem]: 1} : {}}
+      defaultValue={setupMode ? defaultValue : undefined}
```

Comment:
From my understanding yes, because the initialValues only cover the fields that are already in the regform when you load it. If you're in the setup and add a new field it won't have an initial value.. hmm actually now that I'm thinking about it I think we could just rerender the whole form with the new initialValues..